internal/storage/gcs/request.go

```diff
@@ -321,6 +322,23 @@ type Listing struct {
 	ContinuationToken string
 }
 
+// ListingFolders contains a set of folders by a call to ListFolders.
+type ListingFolders struct {
+	// Records for folders matching the listing criteria.
+	Folders []*controlpb.Folder
```

Comment:
should we use our own folder struct here ?

Reply:
Done